installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1520,6 +1541,76 @@ INSERT IGNORE INTO pac_config_key_metadata (`cfkey`,`description`) VALUES ('spri
 INSERT IGNORE INTO pac_config_key_metadata (`cfkey`,`description`) VALUES ('spring.mail.properties.mail.smtp.starttls.enable','Description PlaceHolder');
 INSERT IGNORE INTO pac_config_key_metadata (`cfkey`,`description`) VALUES ('spring.mail.test-connection','Description PlaceHolder');
 
+
+INSERT INTO `pac_config_key_metadata` (`cfkey`, `description`) VALUES('pacman.es.host','Description PlaceHolder');
+INSERT INTO `pac_config_key_metadata` (`cfkey`, `description`) VALUES('pacman.es.port','Description PlaceHolder');
+INSERT INTO `pac_config_key_metadata` (`cfkey`, `description`) VALUES('esLoggingLevel','Description PlaceHolder');
+insert into `pac_config_key_metadata` (`cfkey`, `description`) values('heimdall-host','Description PlaceHolder');
```

Comment:
To be consistent please use **INSERT IGNORE INTO** (caps)

Reply:
Done!!